docs/operator-manual/notifications/index.md

```diff
@@ -45,3 +45,19 @@ kubectl patch app <my-app> -n argocd -p '{"metadata": {"annotations": {"notifica
 ```
 
 Try syncing an application to get notified when the sync is completed.
+
+## Namespace based configuration
```

Comment:
Can you add a description of what happens when there's a "base configuration" defined in the `argocd` namespace and also a cm/secret in the App namespace? What happens if keys in the ConfigMaps/Secrets conflict? Which takes precedence? An example or two might help.